components/public-api/gitpod/experimental/v1/cell_stats.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service CellStatsService {
+    // Retrieves the current user stats
+    rpc GetUserStats(GetUserStatsRequest) returns (GetUserStatsResponse) {}
+}
+
+message GetUserStatsRequest {
+    // The date for which the stats are requested, optional.
+    string date = 1;
+}
+
+message GetUserStatsResponse {
+    // The count of unique concurrent users.
+    uint64 unique_concurrent_users = 1;
```

Comment:
This should be on that particular date. The below `all_time_users` should be since the beginning of time.

Reply:
Agree this would be tricky. We can remove the requirement for capturing `all_time_users` - we only need the unique concurrent users at the time of "asking". I will update the issue. See [internal discussion.](https://gitpod.slack.com/archives/C01LDA28WV6/p1684859731544219?thread_ts=1684839959.449619&cid=C01LDA28WV6)